static/src/javascripts/bootstraps/common.js

```diff
@@ -251,6 +255,28 @@ define([
                 new Clickstream({filter: ['a', 'button']});
             },
 
+            showAdblockMessage: function () {
+                var alreadyVisted = storage.local.get('alreadyVisited') || 0,
```

Comment:
I know about the Message, but this is something different. We just don't want to show that info to anyone who is on our page for the first time.